src/linuxX64Main/kotlin/mu/KMarkerFactory.kt

```diff
@@ -0,0 +1,11 @@
+package mu
+
+/**
+ * A platform independent factory to create markers.
+ */
+actual object KMarkerFactory {
+    actual fun getMarker(name: String): Marker {
+        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
```

Comment:
please remove "To change body of ..." and change todo to UnsupportedOperationException (if you're not going to support it at the moment)

Reply:
What is the marker for. I saw the interfaces for it. But wasn't entirely sure it's use case?

Reply:
We used slf4j markers concept so I think quite a good explanation is here: https://stackoverflow.com/questions/16813032/what-are-markers-in-java-logging-frameworks-and-what-is-a-reason-to-use-them 
If you want to take ideas from other implementations in kotlin-logging take a look at the js implementation as it is similar by the fact that it's not the original api the library was built with.